web/packages/shared/components/Notification/Notification.tsx

```diff
@@ -178,6 +182,16 @@ function getRenderedContent(
   }
 }
 
+function isContentAnObject(
```

Comment:
I feel like this is checking if it's more than just an object and it deserves a better name but I'm drawing a blank here

Reply:
Right, I renamed it to `isContentANotificationItemObject` to match the interface name.